tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -76,3 +82,78 @@ describe('OptionsListUtils', () => {
         await measureFunction(() => OptionsListUtils.getMemberInviteOptions(personalDetails, mockedBetas, SEARCH_VALUE));
     });
 });
+
+/* formatSectionsFromSearchTerm performance tests */
+describe('formatSectionsFromSearchTerm', () => {
```

Comment:
we don't need this lets move all tests inside here https://github.com/Expensify/App/blob/3d780639eb260c1165e884032f4a92aa7dbbb4ca/tests/unit/OptionsListUtilsTest.js#L9